### ISSUE TITLE
Fails to compile with cython 3.x

### ISSUE THREAD

**Issue by lanodan** [renpy/renpy#5359]
Downstream issue (filled for `pygame_sdl2` but renpy has the same errors): https://bugs.gentoo.org/911781

**Comment by renpytom** [comment]
Correct. Until Ren'Py has been updated, please use Cython 2.

**Comment by lanodan** [comment]
Yeah, no problem there, just making sure the renpy project is aware and it's properly tracked rather than lost in a distro bugtracker.

Is there any blockers on it by the way?

**Comment by renpytom** [comment]
No, it's more a "I'll get to it when I get to it" sort of thing. I'm not sure how easy it will be to make the change before I end support for Python 2.7 in May, and so I'm waiting until then.

**Comment by KSJ2000** [comment]
This can be now closed. It builds fine with cython 3.
